### PR TITLE
build-artifacts.sh improvements

### PR DESCRIPTION
**Pull request by crobinso**
Here's some small build-artifacts.sh improvements that helped
me get things building. See individual commits for details

### CODE REVIEW

**Comment by YanVugenfirer** [comment]
@crobinso Hi Cole,

It looks that there are conflicts in PR.

Best regards,
Yan.

**Comment by crobinso** [comment]
I've rebased and force pushed now

File: automation/build-artifacts.sh [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+# It is that the path will be under 260 chars
+# If we get an light.exe error "The system cannot find the file..."
+# This is probably the cause
+VIRTIO_WIN_DRIVERS_PATH=${1:-"$PWD/vwi"}
+
+if [ ! -e "${VIRTIO_WIN_DRIVERS_PATH}" -o \
+     ! -e "${VIRTIO_WIN_DRIVERS_PATH}/viorng" ] ; then
+    echo "Must pass in a directory containing virtio-win iso content. "
+    echo "If using virtio-win-pkg-scripts, this is something like "
+    echo "tmp*/make-driver-dir-output-*"
```
Comment (Gal-Zaidman) [inline automation/build-artifacts.sh]: We want to project to be the same for U/S and D/S so I don't like referencing "virtio-win-pkg-scripts", I suggest changing the error massage

**Comment by Gal-Zaidman** [comment]
Thanks for the PR cole :),
I'm merging because the comment is not that important currently and I wanted you to be able to continue while we are on shutdown

File: automation/build-artifacts.sh (hunk 1 above)
Comment (crobinso) [reply automation/build-artifacts.sh]: Makes sense, feel free to push a commit tweaking the message to what works for you, the virtiio-win-pkg-scripts mentions aren't necessary for my needs

**Pull request merged**